v2/apiserver/internal/core/mongodb/events_store_test.go

```diff
@@ -615,13 +614,45 @@ func TestEventsStoreCancelMany(t *testing.T) {
 					return nil, errors.New("something went wrong")
 				},
 			},
-			assertions: func(events core.EventList, err error) {
+			assertions: func(err error) {
 				require.Error(t, err)
 				require.Contains(t, err.Error(), "error updating events")
 				require.Contains(t, err.Error(), "something went wrong")
 			},
 		},
 
+		{
+			name: "error counting canceled events",
```

Comment:
This whole test case can go away if we don't make the extra db call to count affected events.